rasterfont/README.adoc

```diff
@@ -0,0 +1,15 @@
+# Raster Font Example
+
+This is a simple font bitmap font loader.
+
+With this code you can load fonts stored in pngs and change it's color.
+
+image:https://user-images.githubusercontent.com/292738/124190140-cbd75f00-da8f-11eb-9f80-14bffd4973ea.png[]
+
+Keep in mind this changes each pixel color. So it may not be performant in huge images.
```

Comment:
You probably don't need to do this if you use the DrawTransform "MONO" color mode (assuming the font is always a single color), which might ease the load a bit.

Reply:
I think this specific font uses two colors.  So a single color would not be pretty, that was the reason I have to change it pixel by pixel. I will add the DrawTransform "MONO" to the docs though.

![imagen](https://user-images.githubusercontent.com/292738/160241075-18bed9f1-0618-4f65-90f6-68bdd5e81fd7.png)